app/models/ability.rb

```diff
@@ -4,8 +4,11 @@ class Ability
   def initialize(user)
      user ||= User.new
 
+     alias_action :create, :read, :update, :destroy, :to => :crud
+
       if user.is_admin?
-        can :manage, :all
+        can :crud, User
```

Comment:
Are you sure it will work?
`alias_action` should make an alias for each of this `:create, :read, :update, :destroy` abilities to `:crud`
so, in fact, user has to have only one of these to get admin access.